### ISSUE TITLE
Just curious what are the bbox values in the raw dataset labels?

### ISSUE THREAD

**Issue by zhenglilei** [MichalBusta/E2E-MLT#46]
The MLT format labels have been given. Thanks a lot!

But i am curious what do these floats in the raw label mean? 
0 0.46809631347656255 0.8593862826680818 0.0739853187168 0.0808596013931 -0.0298556635598 منذ

The first number is the language label, e.g., 0 for Arabic, 2 for Chinese...
The 2nd and 3rd number is the center point of the bounding box.
But I have no idea about the others, thanks in advance if someone can clear my question.

**Comment by MichalBusta** [comment]
Lang,centerX,centerY,width,height,angle,transcripion
the width and height are divided by image diagonal.

pá 19. 7. 2019 v 23:10 odesílatel zhenglilei <notifications@github.com>
napsal:

**Comment by ThisIsIsaac** [comment]
@MichalBusta 

I'm still a little confused.

How do we convert centerX, centerY, width, height, and angle to actual boxes?

I have guesses for how to interpret centerX, centerY, width, height:

```python
box_center_x_coord = center_x * image_width
box_center_y_coord = center_y * image_height
box_width = width * image_diagonal
box_height = height * image_diagonal
```

But I have no idea what `angle` means. Is it rotation in radian? Where is the rotation centered?

**Comment by MichalBusta** [comment]
yes, it is rotation in radians centered in the center of bounding box. 
complete code:  
 https://github.com/MichalBusta/E2E-MLT/blob/c147bfbf7fa66c97419d2d38333cbb0f33e91492/data_gen.py#L64
